src/firebase/schemas/index.ts

```diff
@@ -1 +1 @@
-export * from "./Event"
+export * from "../interfaces/Event"
```

Comment:
zakaj si to spremenil?

to spremeni nazaj v
`export * from "./Event"`

Reply:
Error mi vrze ven...

File name '/Users/bezoj/Documents/Summer-of-Trucks/Summer-of-Trucks/src/firebase/schemas/Event.ts' differs from already included file name '/Users/bezoj/Documents/Summer-of-Trucks/Summer-of-Trucks/src/firebase/schemas/event.ts' only in casing.

Reply:
Ja to pa je vscode finta.. spremeni file v `event1.ts` pa pol nazaj v `Event.ts`

Reply:
file je napisan z malo  "event.ts", zato verjento mtalo error. Ke spremenim v vlko zacetnico?

Reply:
imena file-ov se začnejo skos z veliko, to pravilo večina upošteva tak da se tega drži.

Reply:
folderji pa so lowercase